src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -191,27 +192,38 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this.previewFeaturesEnabled = this._configurationService.getValue('workbench.enablePreviewFeatures');
 		this._register(this.themeService.onDidColorThemeChange(this.updateTheme, this));
 		this.updateTheme(this.themeService.getColorTheme());
+		this.updatePreview();
 		this.setFocusAndScroll();
-		this.cellModel.isEditMode = false;
 		this._htmlMarkdownConverter = this._instantiationService.createInstance(HTMLMarkdownConverter, this.notebookUri);
-		this._register(this.cellModel.onOutputsChanged(e => {
+		this._register(this.cellModel.onOutputsChanged(_e => {
```

Comment:
Why the underscore here?

Reply:
vscode was complaining to me that unused variables should have underscores. We don't have a linting rule for this, so it didn't cause the build to fail. Just saw it. Happy to revert, though.

Reply:
Can't you replace the arg with empty parentheses if it's not being used?